stl/inc/xutility

```diff
@@ -3430,8 +3359,7 @@ _NODISCARD constexpr const tuple_element_t<_Index, tuple<_Types...>>&& get(const
 
 namespace ranges {
     // clang-format off
-    // CLASS TEMPLATE ranges::subrange
-    template <class _From, class _To>
+        template <class _From, class _To>

```

Comment:
why did formatting change

Reply:
Because I'm way too reliant on clang-format and forgot there's regions where that's off 🤦‍♂️. Actually just noticed this when removing the latest batch.